tests/verifyAWSRDSMySQLInstance_test.js

```diff
@@ -80,3 +82,40 @@ Scenario(
     }
   },
 );
+
+Scenario(
+  'PMM-T716 - Verify adding PostgreSQL RDS monitoring to PMM via UI @not-pr-pipeline',
```

Comment:
why this test in `Monitoring AWS RDS MySQL DB` feature?

Reply:
I will create new feature. I forgot about this. Thanks! :)